app/react/package.json

```diff
@@ -49,15 +49,15 @@
     "lodash": "^4.17.15",
     "mini-css-extract-plugin": "^0.8.0",
     "prop-types": "^15.7.2",
-    "react-dev-utils": "^9.0.0",
+    "react-dev-utils": "^10.0.0",
     "regenerator-runtime": "^0.13.3",
     "semver": "^6.0.0",
     "ts-dedent": "^1.1.0",
     "webpack": "^4.33.0"
   },
   "devDependencies": {
     "@types/mini-css-extract-plugin": "^0.8.0",
-    "@types/node": "^12.12.11",
+    "@types/node": "^13.1.5",
```

Comment:
Should we lock these to the supported minimum, like 10.x? I didn't check the minimum.